AdminUI/Dockerfile.debug

```diff
@@ -1,5 +1,5 @@
 # Build stage
-FROM node:16-alpine as build-stage
+FROM node:18-alpine as build-stage
```

Comment:
You should use 18.16 here as well, so that the behavior of the `Dockerfile` and `Dockerfile.debug` is similar, and there are not surprises in the deployed version.